src/api/swg-client-event-manager-api.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright 2018 The Subscribe with Google Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AnalyticsEvent,EventOriginator} from '../proto/api_messages';
+
+/** @enum {number}  */
+export const FilterResult = {
+  /** the event is allowed to proceed to the listeners */
```

Comment:
Capitalize.

Reply:
I'm matching the casing from the protos.  Are you referring to the name of the FilterResult enum (which matches the AnalyticsEvent enum)?

Reply:
No. I'm only referring to the comments. "_T_he event is ...." and good to end with a period.